README.md

```diff
@@ -47,36 +44,26 @@ subscribers to those tags via the external services.
 * GOV.UK Notify API key and other details (see
   [`email_service.yml`](config/email_service.yml) for required fields)
 
-### Initial setup
-
-* Check that the configuration in `config/database.yml` is correct
-* Run `bundle exec rake db:setup` to load the database
-
 ### Running the application
 
-```bash
-$ ./startup.sh
-```
-
-* email-alert-api runs on port 3088
-* sidekiq-monitoring for email-alert-api uses 3089
+Using any email address that ends with `@notifications.service.gov.uk`
+will not create a subscriber or a subscription, however will return a `201 Created` response.
```

Comment:
Sorry - I didn't notice this on my last review but this text seems to have appeared inadvertently in this section.

Reply:
Oh that's deliberate: it's in the section about "running the app", and I think it would only be useful here - we should be stubbing stuff in the tests. Underlying aim: put some context around the statement, so its utility is clear. 

Does that make more sense?

Reply:
Right I see what you mean. I find the section reads confusingly because there isn't a command to tell you how to run the application. This contrasts with the "Running the test suite" section which tells you how to do it, whereas this just seems to be a gotcha that you might experience in a particular application scenario. 

It might be clearer if you had a quick instruction on how to run the app and then put this in as an aside. 

For what it's worth I think this information seems rather niche for the prominence it has in the readme, it might be better to add it to the API docs for the related endpoints.

Reply:
Sure, that makes sense. I've added another commit for that. How does it look now?